docs/source/releasing_packages.rst

```diff
@@ -0,0 +1,13 @@
+==================
+Releasing packages
+==================
+
+Rez packages can be built and deployed to the :data:`release_packages_path`
+using the :ref:`rez-release` tool. This tool performs the following actions:
+
+* All :doc:`actions <building_packages>` that the `rez-build` tool performs.
+* Executes all configured :data:`release_hooks`.
+* Executes all configured `release_vcs`.
+
+This tool generally runs the same as :ref:`rez-build` otherwise. For more
```

Comment:
Yes and no. It's true that it's somewhat similar, but there is still differences, for example, see https://rez.readthedocs.io/en/stable/package_definition.html#tests. There might also be other differences. For example, a release might result in stricter file permissions (though I don't remember clearly).

Reply:
Basically, I would like to make it clear that that is advantages to use `rez-release`. Though, I have to say that I can't articulate what the advantages are. In other words, when someone reads this page, what will be their conclusion? Will it be that they don't need `rez-release` since it's similar to rez-build` or will it be confusion as to when to use one vs the other?

Reply:
@JeanChristopheMorinPerso Can you clarify what you differences you're referring to with the `tests` section of the `package_definition` page?

Reply:
You can set `run_on` on a test to `pre_release` for example. This test would only run before a release.